config/locales/doorkeeper.en.yml

```diff
@@ -138,6 +140,7 @@ en:
         notifications: Notifications
         profile: Your Mastodon profile
         push: Push notifications
+        offline_access: Offline access
```

Comment:
These messages appear like this:
<img width="611" alt="Screenshot 2025-04-10 at 21 41 52" src="https://github.com/user-attachments/assets/34ddf2ea-ab74-46a9-ade3-06727826052d" />

("Issue 19890" just happened to be the application in my database I was messing with, it'd normally be something like "Phanpy" or something)